### ISSUE TITLE
False positive DAR101 and DAR102 with raw docstring and variadic arguments.

### ISSUE THREAD

**Issue by MT-0** [terrencepreilly/darglint#167]
With the `test.py` file:

```python
def test_function(*args: int) -> int:
    r"""Test a docstring with variadic arguments.
    
    Args:
        \*args:
            A list of integers.
    
    Returns:
        int: The first number else 0.
    """
    return args[0] if args else 0
```

and the `.darglint` file:

```ini
[darglint]
docstring_style = google
strictness = full
```

Running `darglint test.py` outputs:

```
test.py:test_function:4: DAR101: - *args
test.py:test_function:5: DAR102: + \*args
```

**Comment by terrencepreilly** [comment]
It's because of the backslash.  A backslash isn't necessary, because there's no pattern matching or anything; an '*' is just an '*'.  You should be able to change "\*args" to "*args" or "args" and it ought to work.

**Comment by MT-0** [comment]
This arose from a conflict between the errors raise by `darglint` and `flake8-rst-docstrings`. The above code validates using `flake8-rst-docstrings` but does not in `darglint` and removing the backslash then the reverse is true.

Looking at [https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google) then it seems that the backslash is expected outside the "Args" section (so that `*` is not interpreted as the start of an emphasis block) but there is a special case in the "Args" section where it is not expected.

It looks like there is an [open issue](https://github.com/peterjc/flake8-rst-docstrings/issues/18) in `flake8-rst-docstrings` about this and it needs fixing at their end rather than in this package.
